demo/summer-2016.html

```diff
@@ -211,16 +211,22 @@
 <script src="kitchen-sink/require.js"></script>
 <script>
 require.config({ paths : { "ace" : "../lib/ace" } });
-require(["ace/ace", "ace/ext/language_tools", "ace/ext/papeeria/katex-previewer", "ace/ext/papeeria/highlighter"], 
-  function(ace, langTools, katexPreviewer, highlighter) {
+require([
+        "ace/ace",
+        "ace/ext/language_tools",
+        "ace/ext/papeeria/katex-previewer",
+        "ace/ext/papeeria/highlighter",
+        "ace/ext/papeeria/spellchecker"
+        ],
```

Comment:
pls indent this array in usual way: two-space indent, closing bracket with no indent